src/Symfony/Component/Validator/Tests/Constraints/LengthValidatorTest.php

```diff
@@ -89,12 +89,11 @@ public function getFiveOrMoreCharacters()
         );
     }
 
+    /**
+     * @requires extension mbstring
```

Comment:
Well, but why skipping then if someone has the `iconv` but not the `mbstring` extension?